### PR TITLE
Add Curio parachain

### PR DESCRIPTION
**Pull request by alexxxkny**
- Add Curio parachain runtime-api/type definitions
- Add ui setting for a future testnet

### CODE REVIEW

**Review by jacogr — DISMISSED** [review]
Please merge in master - GH shows conflicts (however doesn't _quite_ indicate where they are...)

File: packages/apps-config/src/api/spec/curio.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Copyright 2017-2023 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+
+/* eslint-disable sort-keys */
+const definitions: OverrideBundleDefinition = {
+    types: [
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/curio.ts:8]: If you run `yarn lint` locally it should just (hopefully) fix the whitespace issues in one go. (Just an issue of using 4 spaces between the 2 as we generally use here)

Comment (alexxxkny) [reply packages/apps-config/src/api/spec/curio.ts]: Done. Thank you

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**